rdr_client/hpo_assigner.py

```diff
@@ -2,7 +2,7 @@
 
 Usage:
   ./run_client.sh --project all-of-us-rdr-prod --account $USER@pmi-ops.org \
-      hpo_assigner.py participant_ids_and_hpos.csv [--dry_run]
+      hpo_assigner.py participant_ids_and_hpos.csv --pairing [site|organization|awardee] [--dry_run]
```

Comment:
Once this is merged, make sure to update the oncall playbook:

https://docs.google.com/document/d/1FkzFyc_neXn4gAQfdaCoe91mogD4frIAOFtXDeDM-sU/edit#heading=h.jyaxpj74c83k